docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
Perfect
El 20/07/2015 19:17, "Leandro Guillén" <notifications@github.com> escribió:

> In docker/Dockerfile
> <https://github.com/telefonicaid/fiware-orion/pull/1069#discussion_r35019166>
> :
>
> > @@ -0,0 +1,15 @@
> > +FROM centos:centos6
>
> @jmcanterafonseca <https://github.com/jmcanterafonseca> We still have 3
> files, a README, a dockerfile, and a docker-compose. I leave it as is now,
> but if you feel it would be better at the root of the project I'll move it.
>
> NTC?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/telefonicaid/fiware-orion/pull/1069/files#r35019166>.
>